superset/app.py

```diff
@@ -342,6 +347,15 @@ def init_views(self) -> None:
             category_label=__("Sources"),
             category_icon="fa-wrench",
         )
+        appbuilder.add_link(
+            "Upload Excel",
+            label=__("Upload Excel"),
+            href="/exceltodatabaseview/form",
+            icon="fa-upload",
+            category="Sources",
+            category_label=__("Sources"),
+            category_icon="fa-wrench",
+        )
```

Comment:
It would be great if this were conditionally added if any of the `EXCEL_EXTENSION` are present in `ALLOWED_EXTENSIONS`. Something like 
```python
if EXCEL_EXTENSION.intersection(ALLOWED_EXTENSIONS):
    appbuilder.add_link(...)
```
Same for CSV.